starter-code/javascript/main.js

```diff
@@ -10,15 +10,17 @@ var milUni      = document.getElementById('milUni');
 
 
 function printTime() {
-setTimeout()
+setTimeout();
 }
 
 function printMinutes() {
-
+minDec.innerHTML = chronometer.setTime[0];
```

Comment:
Lo que estáis haciendo aquí es igualar minDec.innerHTML a una función la posición 0 que es undefined. Para que funcione lo que tenéis que hacer es `chronometer.setTime()` ejecutar la función y luego acceder a la posición 0, `chronometer.setTime()[0]`
Lo mismo para las demás.